src/ui-test/tests/jbang_user_preference.test.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import path = require('path');
+import { WebDriver, VSBrowser, Marketplace, By, EditorView, Workbench, InputBox, BottomBarPanel, TerminalView } from 'vscode-uitests-tooling';
+import * as pjson from '../../../package.json';
+import * as fs from 'fs-extra';
+
+describe('JBang user preference version set test', function () {
```

Comment:
I would maybe change the filename from `jbang_user_preference.test.ts` to `camel_jbang_version.test.ts` it seems to me more precise and it goes hand to hand with already existing `camel_catalog_version.test.ts` WDYT?